bundle/manifests/falcon-helm.clusterserviceversion.yaml

```diff
@@ -0,0 +1,258 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "falcon.crowdstrike.com/v1alpha1",
+          "kind": "FalconSensor",
+          "metadata": {
+            "name": "falcon-helm"
+          },
+          "spec": {
+            "falcon": {
+              "aid": null,
+              "apd": null,
+              "aph": null,
+              "app": null,
+              "assert": null,
+              "billing": null,
+              "cid": null,
+              "feature": null,
+              "memfail_every_n": null,
+              "memfail_grace_period": null,
+              "message_log": null,
+              "provisioning_token": null,
+              "tags": null,
+              "trace": null,
+              "update": null
+            },
+            "node": {
+              "daemonset": {
+                "annotations": {},
+                "labels": {},
+                "updateStrategy": "RollingUpdate"
+              },
+              "enabled": true,
+              "fullnameOverride": "",
+              "image": {
+                "pullPolicy": "Always",
+                "repository": "falcon-node-sensor",
+                "tag": "latest"
+              },
+              "nameOverride": "",
+              "podAnnotations": {},
+              "terminationGracePeriod": 10
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.3.0
+    operators.operatorframework.io/project_layout: helm.sdk.operatorframework.io/v1
+  name: falcon-helm.v0.0.1
+  namespace: placeholder
```

Comment:
```suggestion
  namespace: crowdstrike
```

Reply:
should this be something else, e.g. ``crowdstrike``? Recall another conversation about not setting namespaces within Helm though.

Reply:
The operator-sdk generates and manages the `bundle` directory and all files under it based on what is in the `config` directory. This should probably be fixed under the `config/manifests directory`.